diff --git a/pmb/install/_install.py b/pmb/install/_install.py
index 1921c520..5cbea7b4 100644
--- a/pmb/install/_install.py
+++ b/pmb/install/_install.py
@@ -927,6 +927,7 @@ def install_system_image(
         )
     else:
         layout = None
+        size_boot = 0
     if not args.rsync:
         pmb.install.blockdevice.create(args, size_boot, size_root, size_reserve, split, disk)
         if not split and layout:
